infobip-typescript-generator-extension-annotation-processor/src/main/java/com/infobip/typescript/TypescriptAnnotationProcessor.java

```diff
@@ -81,6 +94,18 @@ private TypeScriptFileGenerator createTypeScriptFileGenerator(TypeElement elemen
         }
     }
 
+    @SuppressWarnings("unchecked")
+    private Class<? extends Annotation> getCustomValidationClass(TypeMirror typeMirror) {
+
+        try {
+            Class<?> aClass = getClass().getClassLoader()
```

Comment:
minorna stvar, ali mislim da se cesto u ovim slucajevima varijabla naziva clazz (bar mi zvoni da sam tako nesto vidio po libovima)

Reply:
slažem se ali ja bi to ostavio čisto jer je tako bilo u projektu